docs/pages/router/navigating-pages.mdx

```diff
@@ -65,28 +66,41 @@ To navigate to a route without the stack unwinding, you can use the `push` prop
 
 In contrast, the `replace` method substitutes the current route in the navigation stack with a new one, effectively replacing the current screen with the new one without adding to the stack.
 
-## Imperative navigation
-
-You may want to navigate from a global store when a user logs in or out. You can use the `router` object or `useRouter` hook to navigate imperatively (outside of React).
-
-```js
-import { router } from 'expo-router';
-
-export function logout() {
-  /* @info Navigate to <strong>/login</strong> */
-  router.replace('/login');
-  /* @end */
-}
-```
-
-The `router` object is immutable and contains the following functions:
-
-- **navigate**: `(href: Href) => void` Navigate to a route using the default behavior. You can provide a full path like **/profile/settings** or a relative path like **../settings**. Navigate to dynamic routes by passing an object like `{ pathname: 'profile', params: { id: '123' } }`.
-- **push**: `(href: Href) => void` Navigate to a route by always pushing onto the stack. You can provide a full path like **/profile/settings** or a relative path like **../settings**. Navigate to dynamic routes by passing an object like `{ pathname: 'profile', params: { id: '123' } }`.
-- **replace**: `(href: Href) => void` Navigate to a route by substituting the current route with a new one. This is useful for redirects.
-- **back**: `() => void` Navigate back to previous route.
-- **canGoBack**: `() => boolean` Returns `true` if a valid history stack exists and the `back()` function can pop back.
-- **setParams**: `(params: Record<string, string>) => void` Update the query params for the currently selected route.
+To navigate, you can provide a full path like `/profile/settings`, a relative path like `../settings` or by passing an object like `{ pathname: 'profile', params: { id: '123' } }`.
+
+<Table>
+  <TableHead>
+    <Row>
+      <HeaderCell>Name</HeaderCell>
+      <HeaderCell>Description</HeaderCell>
+    </Row>
+  </TableHead>
+  <tbody>
+    <Row>
+      <Cell>`navigate`</Cell>
+      <Cell>
+        Navigate to a route using the default behavior. Navigate will only push a new screen if the
+        new route is different (not including search parameters or the hash), otherwise the current
+        screen will rerender with the new parameters. If you navigate to a route is that is in the
+        history, the stack will unwind to that route.
+      </Cell>
+    </Row>
+    <Row>
+      <Cell>`push`</Cell>
+      <Cell>
+        The same as `navigate`, except a new screen will always be created. You can push the current
+        route multiple times, or with new parameters.
+      </Cell>
+    </Row>
+    <Row>
+      <Cell>replace</Cell>
+      <Cell>
+        Remove the current route from the history and replace it with the specifed URL. This is
+        useful for redirects
+      </Cell>
+    </Row>
+  </tbody>
+</Table>
```

Comment:
````suggestion
| Name      | Description                                                                                                                               |
| --------- | ----------------------------------------------------------------------------------------------------------------------------------------- |
| `navigate` | Navigate to a route using the default behavior. `navigate` only pushes a new screen if the new route is different (not including search parameters or the hash). Otherwise, the current screen rerenders with the new parameters. If you navigate to a route that is in the history, the stack will reset to that route. |
| `push`    | The same as `navigate`, except a new screen is always created. You can push the current route multiple times or with new parameters. |
| `replace` | Remove the current route from the history and replace it with the specified URL. This is useful for redirects.                              |
````

- Let's use the table syntax
- Fix missing code syntax highlight in `navigate` (start of the second sentence).
- Use present tense
- Instead of using the word "unwind", I'd suggest using "reset" as it was hard to understand initially (unless I'm missing something here, please let me know).
- replace -> `replace` as it was missing code syntax highlight